vl-datepicker.src.js

```diff
@@ -52,10 +63,57 @@ export class VlDatepicker extends VlElement(HTMLElement) {
                 </button>
             </div>
         `);
+        this._value = null;
     }
 
     connectedCallback() {
         this.dress();
+        this._element.querySelector('#input')._flatpickr.config.onChange.push(
```

Comment:
Zetten we dit niet beter bij de `dress` methode, want ik dacht dat `connectedCallback` mogelijk een tweede keer kon uitgevoerd worden (bvb bij DOM move)?

Reply:
Idd, ik zal het verplaatsen